README.md

```diff
@@ -1,35 +1,148 @@
+
+- [File Naming and Organization](#file-naming-and-organization)
+- [Views](#views)
+- [ViewModels](#viewmodels)
+
+##### File Naming and Organization
+
+The presentation layer will make up Views and ViewModels and those are stored in the App/Features folder.  The Features folder attempts to group presentation files by feature type. For example:
+Features/Lessons
+Features/Onboarding
+Features/ToolSettings
+
+Where Lessons, Onboarding, and ToolSettings are app features and falling under each feature directory make up the screens (Views) pertaining to that feature.
+
+Under each Feature folder should be the screens (Views) that make up that feature.  Each screen (View) should be placed in a directory name pertaining to that screen.  For example, the Lessons feature has the following screens (Views) associated with it. Lesson, LessonEvaluation, and LessonList.  There would then be a directory for each screen like so.
+
+Feature/Lessons/Lesson
+Feature/Lessons/LessonEvaluation
+Feature/Lessons/LessonList
+
+Under each screen (View) folder in a feature should be the View and associated ViewModel that make up that screen.  The Lesson screen (View) would then have LessonView and LessonViewModel.
+
+Feature/Lessons/Lesson/LessonView.swift
+Feature/Lessons/Lesson/LessonViewModel.swift
+
+Any smaller view components that help in creating the screen (View) should go in a Subviews directory.  From the above example, if we built some smaller subview components that help make up the LessonView.swift, those would go in the following:
 
-MVVM-C
+Feature/Lessons/Lesson/Subviews
 
-Model - The model layer operates on the data layer. Performs networking, json decoding, and other various data services. 
+If there are any views or subviews that are shared across screens (Views) or shared across Features.  Those should go in the following:
+App/Share/SwiftUI Views
 
-View - The view layer operates on the presentation layer and is very simple in form as it is a visual display.  It makes up the following responsibilities:
-- Rendering.
+##### Views
+
+Views are what a user visually sees and interacts with.
+
+Responsibilities:
+- Rendering logic.
 - Animation logic.
-- Observing viewModel output which triggers re-rendering of the view.
-- Sending user inputs to the viewModel (button tap, entering text input, etc.).
+- Owns a ViewModel.
+- Observes ViewModel output.  In SwiftUI this is a combination of @ObservedObject and @Published properties.
+- Send inputs to the ViewModel (button tap, entering text input, etc.).
+
+File Naming and Organization:
+- All newly created views should be created in SwiftUI.
+- Views should have only 1 ViewModel.
+- Subviews that are static and help make up a screen (View) can point to the parent screen (View) ViewModel or they can have their own ViewModel.
+- Subviews that are dynamic such as views in collections (lists, stacks, etc.) should have there own ViewModel and only 1 ViewModel.
```

Comment:
This is a good distinction, I was going back and forth in a couple places on when to use the parent's viewModel and when to create a separate one.  Also some static subviews are so small that I didn't make a viewModel, just pass in the title (or whatever simple data) directly.

Reply:
Here I prefer static SubViews pointing to the parent (Screen) ViewModel.  It will make for less viewModel layers to jump through and less viewModel layers to pass dependencies through.